src/HomeKit/HMEnums.cs

```diff
@@ -208,7 +208,18 @@ public enum HMCharacteristicType {
 		TargetPosition,
 		[iOS (9,0)]
 		TargetVerticalTilt,
-
```

Comment:
So `[Watch (3,0), TV (10,0), iOS (10,0)]` for all of them, that's quite verbose. I knew it was good for input values but it's a lot of noise.